### PR TITLE
[stable/spring-cloud-data-flow] Update to new SCDF version 2.5.0

### PR DESCRIPTION
**Pull request by chrisjs**
#### What this PR does / why we need it:

- update scdf version to 2.5.0
- update skipper version to 2.4.0
- minor cleanups

Signed-off-by: Chris Schaefer <cschaefer@vmware.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/spring-cloud-data-flow/templates/server-config.yaml [hunk 1]
```diff
@@ -23,6 +23,9 @@ data:
                     limits:
                       memory: {{ .Values.deployer.resourceLimits.memory }}
                       cpu: {{ .Values.deployer.resourceLimits.cpu }}
+{{- if .Values.server.containerConfiguration }}
```
Comment (sabbyanandan) [inline stable/spring-cloud-data-flow/templates/server-config.yaml:26]: Probably a dumb question. Any reason why this is indented differently?

File: stable/spring-cloud-data-flow/values.yaml [hunk 2]
```diff
@@ -37,11 +37,17 @@ server:
   #    cpu: 0.5
   #    memory: 640Mi
   extraEnv: {}
+  containerConfiguration: {}
```
Comment (sabbyanandan) [inline stable/spring-cloud-data-flow/values.yaml:40]: Don't we want to highlight this in the README as an option? Perhaps point to the docs in the Microsite/ref.guide?

File: stable/spring-cloud-data-flow/templates/server-config.yaml (hunk 1 above)
Comment (chrisjs) [reply stable/spring-cloud-data-flow/templates/server-config.yaml:26]: the `indent 10` below formats the toYaml block

File: stable/spring-cloud-data-flow/values.yaml (hunk 2 above)
Comment (chrisjs) [reply stable/spring-cloud-data-flow/values.yaml:40]: README updated for both props

**Review by sabbyanandan — APPROVED** [review]
/lgtm

**Pull request merged**